cmd/buildah/rmi.go

```
@@ -248,19 +259,21 @@ func removeImage(image *storage.Image, store storage.Store) (string, error) {
 	if _, err := store.DeleteImage(image.ID, true); err != nil {
 		return "", errors.Wrapf(err, "could not remove image %q", image.ID)
 	}
-	for parent != nil {
+	if parent != nil {
 		nextParent, err := getParent(store, parent.TopLayer)
 		if err != nil {
-			return "", err
+			logrus.Debugf("unable to get parent from image %q: %v", image.ID, err)
+			return image.ID, nil
```

Don't want to return the image ID, but want to return the error that occurred.